docs/MO_DG/prepare_model/Model_Optimizer_FAQ.md

```diff
@@ -624,10 +624,10 @@ If a `*.caffemodel` file exists and is correct, the error occurred possibly beca
 
 #### 100. What does the message "SyntaxError: 'yield' inside list comprehension" during MxNet model conversion mean? <a name="question-100"></a>
 
-The issue "SyntaxError: `yield` inside list comprehension" might occur during converting MXNet models (mobilefacedet-v1-mxnet, brain-tumor-segmentation-0001) on Windows platform with Python 3.8 environment. This issue is caused by the API changes for `yield expression` in Python 3.8.
+The issue "SyntaxError: `yield` inside list comprehension" might occur during converting MXNet models (`mobilefacedet-v1-mxnet`, `brain-tumor-segmentation-0001`) on Windows platform with Python 3.8 environment. This issue is caused by the API changes for `yield expression` in Python 3.8.
 The following workarounds are suggested to resolve this issue:
 1. Use Python 3.6/3.7 to convert MXNet models on Windows
-2. Update MXNet by using `pip install mxnet=1.7.0.post2`
+2. Update Apache MXNet by using `pip install mxnet=1.7.0.post2`
```

Comment:
```suggestion
2. Update Apache MXNet by using `pip install mxnet==1.7.0.post2`
```
Error in original text

Reply:
No problem. I'm still working on renaming in other files/articles, so I'll regularly update this PR :)